ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/validation/RequestValidatorTest.java

```diff
@@ -257,6 +258,22 @@ public void shouldThrowIfTooManyPersistentQueries() {
         "persistent queries to exceed the configured limit"));
   }
 
+  @Test
+  public void ff() {
```

Comment:
Maybe a more descriptive name for the test?

Reply:
...whoops